build.cake

```diff
@@ -707,7 +707,8 @@ Task("PackageNuGet")
         });
         NuGetPack("nuget/extensions/teamcity-event-listener.nuspec", new NuGetPackSettings()
         {
-            Version = packageVersion,
+		    // The teamcity-event-listener extension uses its own versioning
+            Version = "1.0.0" + dbgSuffix,
```

Comment:
If we are going to be versioning each plugin on it's own update cycle, we should switch each of them to use their own version/AssemblyInfo and extract the version to use here from that. I just did that in a private project with lots of plugins and it worked well. It meant that I only have to update the version in one place. We might even want to do the same for the framework and the engine, pull the common version, rather than update it there and here. Just thinking out loud :smile: